addons/account/views/account_journal_views.xml

```diff
@@ -11,8 +11,14 @@
                     <field name='sequence' widget='handle'/>
                     <field name="name"/>
                     <field name="type"/>
-                    <field name="journal_group_ids" widget="many2many_tags" readonly="1"/>
-                    <field name="company_id" groups="base.group_multi_company"/>
+                    <field name="journal_group_ids" widget="many2many_tags" readonly="1" optional="show"/>
+                    <field name="currency_id" groups="base.group_multi_currency" optional="hide"/>
+                    <field name="code" optional="show"/>
+                    <field name="default_account_id"/>
```

Comment:
no `optional` on this one?

Reply:
It's not required by the task, what's your opinion @tsb-odoo  ?

Reply:
@smetl @FlorianGilbert IMHO : default account should be optional show.